action.go

```diff
@@ -7,7 +7,7 @@ package holochain
 import (
 	"errors"
 	"fmt"
-	. "github.com/holochain/holochain-proto/hash"
+	. "github.com/HC-Interns/holochain-proto/hash"
```

Comment:
I'm not comfortable changing the reference to Hash everywhere to this other repo.  Is there a specific reason for it?